converters/fromZigbee.js

```diff
@@ -1564,22 +1564,43 @@ const converters = {
         cid: 'genOnOff',
         type: 'cmdOn',
         convert: (model, msg, publish, options) => {
-            return {action: 'toggle'};
+            return {action: 'power', state: 'on'};
         },
     },
     tint404011_off: {
         cid: 'genOnOff',
         type: 'cmdOff',
         convert: (model, msg, publish, options) => {
-            return {action: 'toggle'};
+            return {action: 'power', state: 'off'};
```

Comment:
Why not just return `action: 'off`' here (same for on)

Reply:
Therefore was noch reasonable, wenn can so ist with in anderen Off.

Reply:
I don't understand, could you explain this further?